service/application/docker/runEventManagementAgentDocker.sh

```diff
@@ -6,4 +6,4 @@ if [ -z "$1" ]
     exit 1
 fi
 
-docker run -d -p 8180:8180 --name event-management-agent event-management-agent:$1
\ No newline at end of file
+docker run -d -p 8180:8180 -v /tmp/configFiles/perf1-ema.yml:/config/ema.yml --name event-management-agent event-management-agent:$1
```

Comment:
The config file path and name should be a parameter:

```
#!/usr/bin/env bash

if [ -z "$2" ]
  then
    echo "usage: runEMADocker <tag> <configFile>"
    exit 1
fi

docker run -d -p 8180:8180 -v $2:/config/ema.yml --name event-management-agent event-management-agent:$1

Reply:
Yes, agree. Beat me to it.